src/bicep/mlz.bicep

```
@@ -253,54 +253,82 @@ module sharedServicesVirtualNetworkPeering './modules/spokeNetworkPeering.bicep'
 }
 
 module hubPolicyAssignment './modules/policyAssignment.bicep' = {
-  name: '${hubResourceGroupName}-policyAssignement'
+  name: 'deploy-hub-policyAssignement'
```

I want to say these deployment objects at the subscription scope will need to be more unique than `deploy-hub-policyAssignment` -- could be wrong, but I think any second deployment to the same subscription would encounter a deployment naming collision?

Assuming the previous implementation doesn't encounter those as often:

```suggestion
  name: '${hubResourceGroupName}-policyAssignment'
```